src/blant.c

```diff
@@ -659,8 +679,8 @@ static int RunBlantFromGraph(int k, unsigned long numSamples, GRAPH *G)
 	    for(j=0; j<_numConnectedOrbits; j++) {
 		if (k == 4 || k == 5) orbit_index = _connectedOrbits[_orca_orbit_mapping[j]];
 		else orbit_index = _connectedOrbits[j];
-		if (!_MCMC_EVERY_EDGE || (_sampleMethod != SAMPLE_MCMC && _sampleMethod != SAMPLE_NODE_EXPANSION && _sampleMethod != SAMPLE_SEQUENTIAL_CHAINING)) printf(" %.15g", ODV(i,orbit_index));
-		else printf(" %.12f", _doubleOrbitDegreeVector[orbit_index][i]);
+		if (_MCMC_EVERY_EDGE || (_sampleMethod != SAMPLE_MCMC && _sampleMethod != SAMPLE_NODE_EXPANSION && _sampleMethod != SAMPLE_SEQUENTIAL_CHAINING)) printf(" %.15g", ODV(i,orbit_index));
```

Comment:
From what I understood the ! mark was a bug here.
But I doubt I'm wrong. Please check.

Reply:
I'm not sure either.... but in any case the logic is horrible to read. It should probably be "if(samplemethod *IS* MCMC or NBE or EBE or SEC) then print one thing; else print the other. And I don't know where the MCMC_EVERY_EDGE fits in... but we can wait and see what breaks...